styles/_header.scss

```
@@ -12,6 +12,16 @@ $header-height: 71px;
 
 .logo img {
   max-height: 37px;
+  vertical-align: middle;
+}
+
+.logo span {
+  margin: 0 10px 0 10px;
```

nit: we can use the shorthand here

```suggestion
  margin: 0 10px;
```